src/script/lua_api/l_settings.cpp

```diff
@@ -54,6 +56,17 @@ static inline int checkSettingSecurity(lua_State* L, const std::string &name)
 		return -1;
 	}
 
+	const char *disallowed[] = {
+		"main_menu_script", "shader_path", "texture_path", "screenshot_path",
+		"serverlist_file", "serverlist_url", "map-dir", "contentdb_url",
```

Comment:
Is the dash in `map-dir` a typo? (also what about using a hash map instead?)

Reply:
> Is the dash in map-dir a typo?

no https://github.com/minetest/minetest/blob/8b74257bf3cbb54e78614ac6aaaba79adf75cc8e/src/main.cpp#L731

> what about using an unordered set instead?

meh